Jenkinsfile

```diff
@@ -138,46 +148,58 @@ pipeline {
                 }
             }
         }
-        stage('Build image') {
+        stage('Build images') {
             when {
                 branch pattern: "release/[0-9].[0-9].[0-9]", comparator: "REGEXP"
                
             }
             steps {
                 script {
                     withCredentials([usernamePassword(credentialsId: 'docker-login-streetcode', passwordVariable: 'password', usernameVariable: 'username')]){
-                        sh "docker build -t ${username}/streetcode:latest ."
+                        // Build the backend image
+                        sh "docker build -f Dockerfile -t ${username}/streetcode:${env.CODE_VERSION} ."
                         IS_IMAGE_BUILDED = true
+
+                        // Build the dbupdate image
+                        sh "docker build -f Dockerfile.dbupdate -t ${username}/dbupdate:${env.CODE_VERSION} ."
+                        IS_DBUPDATE_IMAGE_BUILDED = true
                     }
                 }
             }
         }
-        stage('Push image') {
+        stage('Push images') {
             when {
-                expression { IS_IMAGE_BUILDED == true }
+                expression { IS_IMAGE_BUILDED == true || IS_DBUPDATE_IMAGE_BUILDED == true }
```

Comment:
We need AND condition here instead of OR, we can't push images if one of them was not built